thingsboard/pombump-deps.yaml

```diff
@@ -1,25 +1,28 @@
 patches:
-  - groupId: com.nimbusds
-    artifactId: nimbus-jose-jwt
-    version: 9.37.2
-  - groupId: com.squareup.okio
-    artifactId: okio
-    version: 3.4.0
-  - groupId: kotlin-stdlib
-    artifactId: kotlin-stdlib
-    version: 1.4.21
-  - groupId: net.minidev
-    artifactId: json-smart
-    version: 2.4.9
-  - groupId: com.squareup.wire
-    artifactId: wire-schema-jvm
-    version: 4.9.9
-  - groupID: com.google.protobuf
-    artifactID: protobuf-java
-    version: 3.25.5
-  - groupId: org.springframework
-    artifactId: spring-web
-    version: 6.1.12
-  - groupId: org.springframework
-    artifactId: spring-context
-    version: 6.1.14
+    - groupId: com.nimbusds
+      artifactId: nimbus-jose-jwt
+      version: 9.37.2
+    - groupId: com.squareup.okio
+      artifactId: okio
+      version: 3.4.0
+    - groupId: kotlin-stdlib
+      artifactId: kotlin-stdlib
+      version: 1.4.21
+    - groupId: net.minidev
+      artifactId: json-smart
+      version: 2.4.9
+    - groupId: com.squareup.wire
+      artifactId: wire-schema-jvm
+      version: 4.9.9
+    - groupId: ""
+      artifactId: ""
+      version: 3.25.5
```

Comment:
We should not have empty groupID and artifactID we should have
```yaml
  - groupID: com.google.protobuf
    artifactID: protobuf-java
    version: 3.25.5
```

Reply:
trying..